lib/lynx/system_out.rb

```diff
@@ -0,0 +1,14 @@
+require 'open3'
+require 'lynx/error'
+
+module Lynx
+  module SystemOut
+    def self.system_out_with_err(command)
```

Comment:
Small nit, but can we name this something like `run`? The repetition of `system_out`/`SystemOut` is not needed.